spec/index.html

```diff
@@ -484,6 +484,23 @@ <h2>Canonicalization</h2>
     "RDF Canonicalization algorithm version 1.0"
     (<abbr title="RDF Canonicalization algorithm version 1.0"><dfn class="export">RDFC-1.0</dfn></abbr>).</p>
 
+  <p><a href="#fig-ca-overview"></a> provides an overview of <a>RDFC-1.0</a>,
+    with steps 1 through 7 corresponding to the various steps described in
+    <a href="#canon-algo-algo"></a>.</p>
+
+  <figure id="fig-ca-overview" style="text-align:center">
+    <!-- Source for this file is at https://docs.google.com/presentation/d/1iA2m9SZ1XtD0zorfWhi1_rZjvqrxYCnSs3kQR_zyUFk -->
+    <object data="ca-overview.svg" style="width: 100%" type="image/svg+xml" aria-describedby="fig-ca-overview-alt">
+      <p id="fig-ca-overview-alt">
+         The image represents an overview of the <a>RDFC-1.0</a> algorithm.</p>
```

Comment:
```suggestion
      <p id="fig-ca-overview-alt">
        The image represents an overview of the <a>RDFC-1.0</a> algorithm.
        The Input Document is deserialized into the Input Dataset
        and Input Blank Node Identifier Map.
        Canonicalization steps 1-6 are executed resulting in
        the Canonicalized Dataset including the
        Input Blank Node Identifier Map and Issued Identifiers Map.
        Step 7 of the Canonicalization algorithm creates
        the canonical n-quads form of the Canonicalized Dataset.</p>
```

Reply:
The question is the visibility. I presume you rely on setting the `...overview-alt` with the visibility turned off, and rely on screen readers or other tools to retrieve the alt text for the case when it is needed. However, this may not be optimal; I do not remember from the top of my head but, unless you use `display:none`, the alt description affects the overall formatting. On the other hands, some screen readers may skip over texts with alt text (I may be wrong with that). We did an internal a11y review with our spec in EPUB (we had several persons in the group with low vision) and the preference went to the approach based on `details`. 

Another approach that was taken in other specs is to gather all the image description to an appendix, and use aria to link to those.

Reply:
I believe that because the element in inside an `object`, it does not appear for readers not using alternative screen readers, but is available for those unable to see the picture. Keeping it inside the `object` element satisfies the accessibility requirements without cluttering the presentation.

In JSON-LD, we did [place alternate text in an appendix](https://www.w3.org/TR/json-ld11/#image-descriptions) with a reference from the image, which is certainly acceptable, but I don't believe is required.

Reply:
Thank you @gkellogg for adding the alt text!
There's still some debate about how to display these, but I believe there's no problem with the text itself, so I have reflected your suggestion in the PR.

Reply:
The current method, as @gkellogg mentioned, embeds the alt text within the `object` element, and since they are not rendered when the SVG image is displayed, I don't believe they unnecessarily reduce readability.
Also, since the alt text is referred to by `aria-describedby`, I expect it to be properly read out by screen readers that support WAI-ARIA... However, to be honest, I'm not very familiar with the behavior of ARIA or screen readers, so I can't be fully confident here.

Reply:
Ok. I am o.k. leaving it as is; at some point we may get another WAI review (getting close to PR) and we may come back to this then.